agent/service/v1/agent-health-check.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+package io.spoud.sdm.agent.v1;
+
+option java_multiple_files = true;
+option java_package = "io.spoud.sdm.agent.v1";
+option go_package = "agent.v1";
+
+import "agent/domain/v1/domain.proto";
+import "global/selection/v1/pagination.proto";
+import "google/protobuf/wrappers.proto";
+import "global/domain/v1/domain.proto";
+import "google/protobuf/empty.proto";
+
+
+service AgentHealthCheckService {
+  rpc reportEvent (ReportEventRequest) returns (google.protobuf.Empty);
+  rpc reportMetric (ReportMetricRequest) returns (google.protobuf.Empty);
+  rpc reportError (ReportErrorRequest) returns (google.protobuf.Empty);
+}
+
+message ReportEventRequest {
+  io.spoud.sdm.agent.domain.v1.AgentRef agent_ref = 1;
+  io.spoud.sdm.agent.domain.v1.AgentEventType event_type = 2;
+  map<string, string> properties = 3;
```

Comment:
what do you have in mind for properties? generic field to add everything 😅

Reply:
yeah, we have those on most of the entities, I thought it makes sense to have them here too. Even if we don't use them. We could for example send the version or IDK.